dashboards/rss_reader.sp

```diff
@@ -0,0 +1,462 @@
+dashboard "rss_reader" {
+
+  title         = "RSS Reader"
+  documentation = file("./dashboards/docs/rss_reader.md")
+
+  tags = merge(local.rss_common_tags, {
+  type = "Report"
```

Comment:
```suggestion
    type = "Dashboard"
```

Please adjust formatting and I think `Dashboard` is a more suitable category based on the recent changes